spec/factories/habit_logs.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :habit_log do
+    scheduled_at { "2022/02/02" }
```

Comment:
It would probably be better to use actual Date and Datetime objects in these factories. I noticed below you're instantiating a new habit_log factory and passing in a Date object instead of a string. It's important to be consistent as far as types go.